web/server/entrypoint.go

```diff
@@ -146,7 +146,7 @@ func RunServer(ctx context.Context, args []string, _ golog.Logger) (err error) {
 	// This is to ensure we make our best effort to write logs for failures loading the remote config.
 	if cfgFromDisk.Cloud != nil && (cfgFromDisk.Cloud.LogPath != "" || cfgFromDisk.Cloud.AppAddress != "") {
 		var closer func()
-		logger, closer, err = addCloudLogger(logger, rdkLogLevel, cfgFromDisk.Cloud)
+		logger, closer, err = addCloudLogger(logger, logConfig.Level, cfgFromDisk.Cloud)
```

Comment:
Is the cloud logger sensitive to config `debug` changes now, too? I'm guessing not based on the current design, but that does mean that the "Logs" tab in app will not suddenly have debug logs if you add "debug: true" to the config.

Reply:
Another limitation here is that modules that have already been started will not suddenly have debug logging if the RDK config is modified to have it. Note that users _can_ reconfigure a module to have "log_level: debug" to dynamically enable debug logging on a module.

Reply:
> I expect it would be.

I can't remember why I thought this wouldn't apply to the net logger. If you could just verify that debug logs show up in the `Logs` tab after changing the config to have `debug`, that'd be great. 

> Agreed on that limitation. Just double-checking: modules "log" by simply writing to stdout?

The RDK is only sensitive to what a module puts in STDOUT (which it gathers and prefixes with `\TAB`).

> And there's currently no "API" for telling a module to change what it logs? If something like that exists today I'll look into propagating the information to running modules.

You can indeed change the "log level" of a module by reconfiguring it ("log levels" of modules are really just the value passed as `--log-level` when starting the executable). There's no endpoint to change log levels on modules; "reconfiguring" means restarting them with a different `--log-level` flag and adding back the old resources. I don't think it would be a bad idea to do that. Happy to follow up offline on that if you're interested.